grid-view.ios.ts

```diff
@@ -39,6 +40,17 @@ export class GridView extends GridViewBase {
     private _preparingCell: boolean = false;
     private _map: Map<GridViewCell, View>;
 
+    private _orientation: GridViewOrientation = "vertical";
+    public set orientation(orientation: GridViewOrientation) {
```

Comment:
Can't we use `[orientationProperty.setNative]` here like all other properties instead of this normal property?

Reply:
Also lets add the `orientation` property to the `.d.ts` file.